packages/admin/cms-admin/src/userPermissions/user/permissions/ContentScopeGridToolbar.tsx

```diff
@@ -0,0 +1,58 @@
+import { Select } from "@comet/admin-icons";
+import { Button, styled } from "@mui/material";
+import { GridToolbarContainer, GridToolbarFilterButton, GridToolbarQuickFilter } from "@mui/x-data-grid";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+type Props = {
+    actions?: { deleteSelected: () => void; openDialog: () => void };
+};
+
+export const ContentScopeGridToolbar = ({ actions }: Props) => {
+    return (
+        <GridToolbar>
+            <GridToolbarAction>
+                <CustomToolbarAction>
+                    <GridToolbarQuickFilter />
+                    <FilterButton onResize={undefined} onResizeCapture={undefined} />
+                </CustomToolbarAction>
+                {actions ? (
+                    <CustomToolbarAction>
+                        {/* TODO: Add Context Menu once https://github.com/vivid-planet/comet/pull/2115 is merged
+                        <Button variant="text" color="info" endIcon={<MoreVertical />} onClick={actions.deleteSelected}>
+                            <FormattedMessage id="comet.contentScopes.more" defaultMessage="More" />
+                        </Button> */}
```

Comment:
Your PR should depend on this PR if it's needed. Adding functionality that can't work doesn't help in review